attach-asana-task/action.yaml

```
@@ -4,17 +4,20 @@ description: Attaches the current PR to an asana task, if a task can be identifi
 inputs:
   asana-secret:
     description: The secret to use for the Asana API
-    required: true
+    default: ""
+    required: false
 
 runs:
   using: "composite"
 
   steps:
     - name: Create pull request attachments
+      if: inputs.asana-secret != ''
```

Doesn't this need to be changed to 

```suggestion
      if: "${{ inputs.asana-secret }} != ''"
```

---

Nope https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idif
> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.